src/main/java/org/opensearch/knn/index/query/KNNQueryFactory.java

```diff
@@ -80,12 +80,7 @@ public static Query create(CreateQueryRequest createQueryRequest) {
         if (KNNEngine.getEnginesThatCreateCustomSegmentFiles().contains(createQueryRequest.getKnnEngine())) {
             if (filterQuery != null && KNNEngine.getEnginesThatSupportsFilters().contains(createQueryRequest.getKnnEngine())) {
                 log.debug(
-                    String.format(
-                        "Creating custom k-NN query with filters for index: %s \"\", field: %s \"\", " + "k: %d",
-                        indexName,
-                        fieldName,
-                        k
-                    )
+                    String.format("Creating custom k-NN query with filters for index: %s , field: %s , k: %d", indexName, fieldName, k)
```

Comment:
should we add locale as first param? I think this should fail without it

Reply:
seems like it is enforced. I will remove this from my change.